docs/data-sources/cce_node_pool.md

```diff
@@ -0,0 +1,95 @@
+---
+subcategory: "Cloud Container Engine (CCE)"
+---
+
+# huaweicloud\_cce\_node\_pool
+
+To get the specified node pool in a cluster.
+
+## Example Usage
+
+```hcl
+variable "cluster_id" { }
+variable "node_pool_name" { }
+
+data "huaweicloud_cce_node_pool" "node_pool" {
+  cluster_id = var.cluster_id
+  name       = var.node_pool_name
+}
+```
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) The region in which to obtain the cce node pools. If omitted, the provider-level region will be used.
```

Comment:
The prefix description of argument reference start with `Specifies ...`  and atributes reference does not.

Reply:
fixed